### ISSUE TITLE
Crashes in 1.19.1-pre2 when clicking on a mod in the menu

### ISSUE THREAD

**Issue by TheOtterlord** [Prospector/ModMenu#467]
At first I thought it was my mod, but I've also tried it with no mods, and with mods that I know play nicely.

```
---- Minecraft Crash Report ----
// This doesn't make any sense!

Time: 04/07/2022, 11:26
Description: mouseClicked event handler

java.lang.NoSuchFieldError: field_2054
	at com.terraformersmc.modmenu.gui.widget.ModListWidget.select(ModListWidget.java:71)
	at com.terraformersmc.modmenu.gui.widget.entries.ModListEntry.method_25402(ModListEntry.java:85)
	at com.terraformersmc.modmenu.gui.widget.ModListWidget.method_25402(ModListWidget.java:250)
	at net.minecraft.class_4069.method_25402(class_4069.java:27)
	at net.minecraft.class_312.method_1611(class_312.java:94)
	at net.minecraft.class_437.method_25412(class_437.java:490)
	at net.minecraft.class_312.method_1601(class_312.java:94)
	at net.minecraft.class_312.method_22686(class_312.java:165)
	at net.minecraft.class_1255.execute(class_1255.java:103)
	at net.minecraft.class_312.method_22684(class_312.java:165)
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:43)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3474)
	at com.mojang.blaze3d.systems.RenderSystem.limitDisplayFPS(RenderSystem.java:221)
	at net.minecraft.class_310.method_1523(class_310.java:1205)
	at net.minecraft.class_310.method_1514(class_310.java:765)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at com.terraformersmc.modmenu.gui.widget.ModListWidget.select(ModListWidget.java:71)
	at com.terraformersmc.modmenu.gui.widget.entries.ModListEntry.method_25402(ModListEntry.java:85)
	at com.terraformersmc.modmenu.gui.widget.ModListWidget.method_25402(ModListWidget.java:250)
	at net.minecraft.class_4069.method_25402(class_4069.java:27)
	at net.minecraft.class_312.method_1611(class_312.java:94)
	at net.minecraft.class_437.method_25412(class_437.java:490)
	at net.minecraft.class_312.method_1601(class_312.java:94)
	at net.minecraft.class_312.method_22686(class_312.java:165)
	at net.minecraft.class_1255.execute(class_1255.java:103)
	at net.minecraft.class_312.method_22684(class_312.java:165)
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:43)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3474)

-- Affected screen --
Details:
	Screen name: com.terraformersmc.modmenu.gui.ModsScreen
Stacktrace:
	at net.minecraft.class_437.method_25412(class_437.java:490)
	at net.minecraft.class_312.method_1601(class_312.java:94)
	at net.minecraft.class_312.method_22686(class_312.java:165)
	at net.minecraft.class_1255.execute(class_1255.java:103)
	at net.minecraft.class_312.method_22684(class_312.java:165)
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:43)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3474)
	at com.mojang.blaze3d.systems.RenderSystem.limitDisplayFPS(RenderSystem.java:221)
	at net.minecraft.class_310.method_1523(class_310.java:1205)
	at net.minecraft.class_310.method_1514(class_310.java:765)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: Default, Fabric Mods
Stacktrace:
	at net.minecraft.class_6360.method_36565(class_6360.java:49)
	at net.minecraft.class_310.method_1587(class_310.java:2339)
	at net.minecraft.class_310.method_1514(class_310.java:784)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- System Details --
Details:
	Minecraft Version: 1.19.1 Pre-release 2
	Minecraft Version ID: 1.19.1-pre2
	Operating System: Linux (amd64) version 5.17.15-1-MANJARO
	Java Version: 17.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 1209391976 bytes (1153 MiB) / 1811939328 bytes (1728 MiB) up to 2147483648 bytes (2048 MiB)
	CPUs: 8
	Processor Vendor: GenuineIntel
	Processor Name: 11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz
	Identifier: Intel64 Family 6 Model 140 Stepping 1
	Microarchitecture: unknown
	Frequency (GHz): 2.40
	Number of physical packages: 1
	Number of physical CPUs: 4
	Number of logical CPUs: 8
	Graphics card #0 name: TigerLake-LP GT2 [Iris Xe Graphics]
	Graphics card #0 vendor: Intel Corporation (0x8086)
	Graphics card #0 VRAM (MB): 256.00
	Graphics card #0 deviceId: 0x9a49
	Graphics card #0 versionInfo: unknown
	Virtual memory max (MB): 15944.19
	Virtual memory used (MB): 8527.55
	Swap memory total (MB): 0.00
	Swap memory used (MB): 0.00
	JVM Flags: 8 total; -Xss1M -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Launched Version: fabric-loader-0.14.8-1.19.1-pre2
	Backend library: LWJGL version 3.3.1 SNAPSHOT
	Backend API: Mesa Intel(R) Xe Graphics (TGL GT2) GL version 4.6 (Core Profile) Mesa 22.1.2, Intel
	Window size: 2256x1504
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Graphics mode: fast
	Resource Packs: Fabric Mods
	Current Language: English (US)
	CPU: 8x 11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz
```

**Comment by TheOtterlord** [comment]
This crash report is from the run with no other mods installed

**Comment by jackassmc** [comment]
Please post the complete crash report.

**Comment by jackassmc** [comment]
Nevermind, field_2054 was a static NarratorManager instance which was removed in 1.19.1-pre2 (see [NarratorManager.mapping](https://github.com/FabricMC/yarn/commit/9548410a6531af0dea27c92734aaba22ec70a12d#diff-e94cf1a4f5c319187d6e6a8f36f08f2bddda07a4feab508ca04d70fb9c053b89)).

**Comment by jackassmc** [comment]
Fixed in [v4.0.2](https://modrinth.com/mod/modmenu/version/4.0.2), thanks for reporting!
